src/routes/index.ts

```diff
@@ -13,11 +14,16 @@ export function registerRoutes(router: Router): void {
   const cachedR2Middleware = cached(r2Middleware);
   const originMiddleware = new OriginMiddleware();
 
+  const corepackRedirectMiddleware = new RedirectionMiddleware('https://github.com/nodejs/corepack');
```

Comment:
Are the corepack docs available anywhere else?

Reply:
+1